code/game/machinery/vending.dm

```diff
@@ -610,6 +629,7 @@
 					/obj/item/weapon/reagent_containers/food/drinks/soda_cans/cola = 8, /obj/item/weapon/reagent_containers/food/drinks/soda_cans/sodawater = 15,
 					/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 30,/obj/item/weapon/reagent_containers/food/drinks/ice = 9)
 	contraband = list(/obj/item/weapon/reagent_containers/food/drinks/tea = 10)
+	syndicate = list(/obj/item/weapon/reagent_containers/food/drinks/bottle/hooch = 10, /obj/item/weapon/reagent_containers/food/drinks/bottle/hippie = 10)

```

Comment:
both are alcohol which bartender can easily make,hooch is a strong alcohol made with ethanol,welding fuel and sugar, while hippie delight is a normal alcohol like pan galactic gargle blaster, all i did was creating bottles with them in it